spec/forms/placements/placements/filter_form_spec.rb

```diff
@@ -110,19 +110,19 @@
 
     context "when removing partner school id params" do
```

Comment:
We might want to reword the tests and context blocks slightly here to reflect that we're no longer dealing with "partner school IDs":

```suggestion
    context "when removing the partner schools filter" do
```